README.md

```diff
@@ -0,0 +1,27 @@
+# AGA Ratings Program
+
+This program scans the AGA Database for tournaments that have been added but not yet rated.
+
+It is intended to be run after the tournaments have been uploaded into the database.
+
+Compiling the code creates several binaries:
+
+**TODO: document the binaries and what each one is used for**
+
+## Building and Running with Docker
+
+To build and run with the provided Dockerfile, first copy `db_passwords.example.h` to `db_passwords.h` and fill in the values.
+
+Now run:
+
+```
+docker build -t agaratings .
+```
+
+Run with:
+
+```
+docker run --rm -it agaratings <binary-to-execute>
+```
+
+e.g., to check for tournaments that haven't been rated yet, `docker run --rm -it agaratings check`
```

Comment:
I saw the ToDo on line 9. Is there a plan to resolve the ToDo on line 9 in this pull request or a later pull request?

Reply:
I think part of the ToDo is the documents ive created in a shared google folder to explain how to run the ratings application after it has been configured

Reply:
@vash3g Yeah I think it's just also useful to have it documented here with the code. Much easier for volunteers to understand the code's basic workflow that way. :)

@EarthSchlange I was going to do it in a future PR, but with the above comments, now I'm going to do it in this PR.